assets/global.js

```diff
@@ -935,6 +936,11 @@ class VariantSelects extends HTMLElement {
         if (inventoryDestination) inventoryDestination.classList.toggle('visibility-hidden', inventorySource.innerText === '');
 
         this.toggleAddButton(!this.currentVariant.available, window.variantStrings.soldOut);
+
+        // Update quantity form
+        const quantityFormCurrent = document.getElementById(`Quantity-Form-${sectionId}`);
+        const quantityFormUpdated = html.getElementById(`Quantity-Form-${sectionId}`);
+        if (quantityFormCurrent && quantityFormUpdated) quantityFormCurrent.innerHTML = quantityFormUpdated.innerHTML;
       });
   }
```

Comment:
I agree we shouldnt show it at the bottom because the qty rules change depending on the variant, but the information is still important. What about in here?

https://screenshot.click/26-49-mmpox-2ng3f.png

<img width="1185" alt="Screenshot 2023-01-26 at 9 50 40 AM" src="https://user-images.githubusercontent.com/19962891/214867343-f8988797-dae3-4702-b7ce-30cd0b69ca50.png">

Reply:
Good idea! Updated - https://github.com/Shopify/dawn/pull/2251/commits/76ba1b30123cdadb4cf9168c35119a708a89e025

Reply:
Also noticed that [this commit](https://github.com/Shopify/dawn/commit/a422c45be43027628f02c77f8ae98f7c6d34b88c) broke no-js add-to-cart. Fixed this issue - https://github.com/Shopify/dawn/pull/2251/commits/e18fb6131a61d9507e0750e4acacf076a73e02e9